wire-desktop/PKGBUILD

```diff
@@ -3,25 +3,24 @@
 
 pkgname=wire-desktop
 pkgver=3.5.2881
-pkgrel=1
-pkgdesc='Modern, private messenger. Based on Electron.'
-arch=('x86_64' 'i686')
+pkgrel=2
+pkgdesc='End-to-end encrypted messenger with file sharing, voice calls and video conferences'
+arch=('x86_64')
 url='https://wire.com/'
 license=('GPL3')
-conflicts=('wire-desktop-bin')
-depends=('alsa-lib' 'gconf' 'gtk2' 'libxss' 'libxtst' 'nss' 'xdg-utils')
+depends=('alsa-lib' 'electron' 'gconf' 'gtk2' 'libxss' 'libxtst' 'nss' 'xdg-utils')
 makedepends=('cargo' 'npm' 'python2' 'git' 'yarn')
-optdepends=('hunspell-en: for English spellcheck support'
-            'noto-fonts-emoji: for colorful emoji made by Google'
-            'ttf-emojione: for colorful emoji made by EmojiOne')
-provides=('wire-desktop')
-source=("${pkgver}.tar.gz::https://github.com/wireapp/wire-desktop/archive/linux/$pkgver.tar.gz"
+optdepends=('hunspell-en_US: for English spellcheck support'
+            'emoji-font: colorful emoji')
+source=("${pkgname}-${pkgver}.tar.gz::https://github.com/wireapp/${pkgname}/archive/linux/$pkgver.tar.gz"
+        "${pkgname}"
         "${pkgname}.desktop")
 sha256sums=('4216cd9c3a2c4920aec2f3c967181b04bfafdb1b47e526a8e823911cce704da1'
+            '13f0829f0995269e7bdf5db683d048d36ae8af8cf16030a581e6386d83aae03a'
             'cc9056cecff2aa49a9ce9c8376d57ec8c7c2cb8174f7966b5cdccbeb2e3751ea')
 
 build() {
-  cd "${srcdir}/${pkgname}-linux-${pkgver}"
+  cd "${pkgname}-linux-${pkgver}"
   yarn
   yarn build:ts
   npx grunt 'linux-other'
```

Comment:
This task besides compiling a big binary with bundled electron actually prepares the `wrap` directory. We run this task, but then only take the contents of `wrap/dist/linux-unpacked/resources/app/`, essentially ignoring the binary with bundled electron.